spaceros/Dockerfile

```diff
@@ -61,6 +61,7 @@ RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/r
 
 # Install required software development tools and ROS tools (and vim included for convenience)
 RUN apt-get update && apt-get install -y \
+  bison \
```

Comment:
I don't see a reference to why bison is now required in the patch notes. What has made bison required now?

Reply:
Yes, missed that. Updated the initial comment to add the info.